Frameworks/jdi-core/src/main/java/com/epam/jdi/uitests/core/aop/ActionProcessor.java

```diff
@@ -117,71 +172,124 @@ static String methodNameTemplate(MethodSignature method) {
             return Switch(template).get(
-                Case(t -> t.contains("{0"), t -> MessageFormat.format(t, joinPoint.getArgs())),
-                Case(t -> t.contains("{"), t -> {
-                    MapArray obj = new MapArray<>("this", getElementName(joinPoint));
-                    return getActionName(method, t, obj, methodArgs(joinPoint, method), classFields(joinPoint));
-                }),
-                Case(t -> t.contains("%s"), t -> format(t, joinPoint.getArgs())),
-                Default(t -> {
-                    MapArray<String, Object> args = methodArgs(joinPoint, method);
-                    if (args.size() == 1 && args.get(0).value.getClass().isArray())
-                        return format("%s(%s)", t, arrayToString(args.get(0).value));
-                    MapArray<String, String> methodArgs = args.toMapArray(Object::toString);
-                    String stringArgs = Switch(methodArgs.size()).get(
-                        Value(0, ""),
-                        Value(1, v->"("+methodArgs.get(0).value+")"),
-                        Default(v->"("+methodArgs.toString()+")")
-                    );
-                    return format("%s%s", t, stringArgs);
-                })
+                    Case(t -> t.contains("{0"), t -> MessageFormat.format(t, joinPoint.getArgs())),
+                    Case(t -> t.contains("{"), t -> {
+                        MapArray obj = new MapArray<>("this", getElementName(joinPoint));
+                        return getActionName(method, t, obj, methodArgs(joinPoint, method), classFields(joinPoint));
+                    }),
+                    Case(t -> t.contains("%s"), t -> format(t, joinPoint.getArgs())),
+                    Default(t -> {
+                        MapArray<String, Object> args = methodArgs(joinPoint, method);
+                        if (args.size() == 1 && args.get(0).value.getClass().isArray())
+                            return format("%s(%s)", t, arrayToString(args.get(0).value));
+                        MapArray<String, String> methodArgs = args.toMapArray(Object::toString);
+                        String stringArgs = Switch(methodArgs.size()).get(
+                                Value(0, ""),
+                                Value(1, v -> "(" + methodArgs.get(0).value + ")"),
+                                Default(v -> "(" + methodArgs.toString() + ")")
+                        );
+                        return format("%s%s", t, stringArgs);
+                    })
             );
         } catch (Exception ex) {
             throw new JDIUIException("Surround method issue: " +
                     "Can't get action name: " + ex.getMessage());
         }
     }
+
+    /**
+     * Get String of comma-separated elements
+     *
+     * @param array {@code Object[]} array
+     * @return String of array's elements comma-separated
+     */
     static String arrayToString(Object array) {
-        String result = "";
```

Comment:
Where did that go?

Reply:
These methods are doing same thing, but mine is shorter and easier for understanding